io.github.ec_.Quake3e.OpenArena.yaml

```diff
@@ -0,0 +1,73 @@
+app-id: io.github.ec_.Quake3e.OpenArena
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+# CHANGED: Let it look up the command via $PATH
+command: launch
```

Comment:
I suggest naming it `openarena`.

Reply:
Yes. Please do. I have a script I use to generate CLI launchers that aren't as hostile to tab completion as reverse DNS names and, while it *will* do stuff like lowercasing the name and stripping suffixes like `.sh` and `.py`, it's not magic.

Whenever you use a `command` name like `launch`, I have to hard-code a manual override for it.